### PR TITLE
Fix votes received

### PR DESCRIPTION
**Pull request by hunglmtb**
refer #253

image

### CODE REVIEW

File: src/shared/common/number-util.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    )
+    .join(".");
+}
+
+export function numberFromCommaString(n: string): string {
+  return n.toString().replace(/\,/g, "");
+}
+
+
+export const secondsToDuration = (second: number|string) => {
```
Comment (guo) [inline src/shared/common/number-util.ts:17]: https://github.com/iotexproject/web-native-staking-v2/blob/0881de769d055ba0989d68d60d2fbc8824ae2291/src/shared/delegate/panes/votes-received-table.tsx#L101
let's combine the two.

File: src/shared/delegate/panes/votes-received-table.tsx [hunk 2]
```diff
@@ -74,43 +94,33 @@ export class VotesReceivedTable extends PureComponent<Props, State> {
...
+            epochCount: 1,
+            delegateName: registeredName,
+            pagination: { skip: 0, first: 500 }
+          },
+        })
+        .then(({ data: { delegate: {bucketInfo:{bucketInfoList}} } }) => {
+          if(bucketInfoList && bucketInfoList.length > 0){
+            const csvSource = bucketInfoList[0].bucketInfo;
+            csvSource.forEach((bucket: Buckets) => {
+              bucket.voterIotexAddress = bucket.voterIotexAddress.replace("0x", "")
```
Comment (guo) [inline src/shared/delegate/panes/votes-received-table.tsx:113]: does this one have 0x prefix still?

File: translations/en.yaml [hunk 3]
```diff
@@ -146,6 +146,7 @@ delegate.votesreceived: 'Votes Received'
 delegate.ranking: Rank
 delegate.vote_share: 'Vote & Share Campaign'
 delegate.tweet.summary.title: '${delegate} (Retweet or Post to Enter Lucky Draw)'
+delegate.annotate.non-official: '* This is not a node hosted by IoTeX Official Team.'
```
Comment (guo) [inline translations/en.yaml:149]: where is it used?

File: src/shared/delegate/panes/votes-received-table.tsx [hunk 4]
```diff
@@ -158,79 +168,143 @@ export class VotesReceivedTable extends PureComponent<Props, State> {
...
-                      {t("delegate.votesreceived.download")}
-                    </Button>
-                    <CommonMargin />
-                  </Flex>
+        {
+          // @ts-ignore
+          <Query
+            ssr={false}
+            query={GET_CHAIN_META}
+            client={explorerApolloClient}
```
Comment (guo) [inline src/shared/delegate/panes/votes-received-table.tsx:196]: can this be queried from analytics directly?

File: src/shared/delegate/panes/votes-received-table.tsx (hunk 2 above)
Comment (hunglmtb) [reply src/shared/delegate/panes/votes-received-table.tsx:113]: not sure. webbp replace this before return, so I add this for sure no 0x display

Comment (hunglmtb) [reply src/shared/delegate/panes/votes-received-table.tsx:87]: https://github.com/iotexproject/web-native-staking-v2/blob/f5b01bd53d1bd0cd2dde186bfd10f4dbffd47c2a/src/shared/delegate/panes/votes-received-table.tsx#L85
it removes 0x prefix.

File: src/shared/delegate/panes/votes-received-table.tsx (hunk 4 above)
Comment (hunglmtb) [reply src/shared/delegate/panes/votes-received-table.tsx]: updated, use antenna to query

File: src/shared/delegate/panes/votes-received-table.tsx [hunk 5]
```diff
@@ -74,118 +68,141 @@ export class VotesReceivedTable extends PureComponent<Props, State> {
...
                 message: "Error",
                 description: `failed to get votes recieved: ${error.message}`,
                 duration: 3,
               });
-              return null;
-            }
-            const { buckets = [] } = data || {};
-            if (buckets.length > 0) {
-              this.getUpdatedBucket(buckets);
+              return <></>;
```
Comment (guo) [inline src/shared/delegate/panes/votes-received-table.tsx:200]: just</> ?

Comment (hunglmtb) [reply src/shared/delegate/panes/votes-received-table.tsx:200]: it's not supported

**Pull request merged**